### ISSUE TITLE
More specific editorial roles

### ISSUE THREAD

**Issue by fredgibbs** [programminghistorian/jekyll#171]
Would they help us stay organized and manage our workflow?

**Comment by acrymble** [comment]
I support this. I think it clarifies a lot of things for us, but also for people wondering who to contact. I'd also like it if it were clearer on the Project Team page how many submissions I'm willing to work on at once, and if I'm at that capacity (which I currently am).

**Comment by ianmilligan1** [comment]
I support this as well - let's talk about it on our next call. We can have some rotation built in, too.

**Comment by acrymble** [comment]
Action everyone: Add to this ticket your 'role' - what you do for the PH. These will ultimately go on the website project team area.

**Comment by wcaleb** [comment]
I mainly do website design and GitHub Pages support.

**Comment by acrymble** [comment]
editorial role: data management & curation; introductory lessons of core skills.
administrative roles: editorial board meeting chair; broadening access;

**Comment by ianmilligan1** [comment]
editorial role: text analysis and data mining, network analysis, intermediate to advanced lessons.

administrative roles: primary contact person for lesson submissions, keeping an eye on author submissions, co-ombudsperson

**Comment by ahegel** [comment]
Editorial interests: text analysis, data visualization

Administrative interests: design of website, editing lessons

**Comment by miriamposner** [comment]
Editorial interests: data visualization, data manipulation, web design, core skills

Administrative roles: outreach, community management

**Comment by jerielizabeth** [comment]
editorial interests: text analysis, network analysis, data visualization, accessibility, UI/UX, workflows

administrative roles: editing lessons, tbd

**Comment by acrymble** [comment]
should we add these to our profiles, or will that also affect our author statements for those of us who have written tutorials?

**Comment by wcaleb** [comment]
It will only change the author statements if we include them in the "bio" field,  but you could create a new YAML field like "roles" and put the information there. I could then figure out how best to display that new data on the Project Team page.

**Comment by fredgibbs** [comment]
I tried to consolidate and standardize the various roles at https://github.com/programminghistorian/jekyll/wiki/Editorial-Roles, where everyone should modify it however they'd like--I just tried to produce a rough draft we could work from. One caveat: although it's great that we all do a bit of everything, if we list too many people under each role, that starts to defeat the point of the list, which is to delegate efficiently. 

For administrative roles, there's something to be said for having a primary person for each task, but that might be a topic for the next call.

For editorial topics, I'm hoping we can eventually migrate this to a more public page on the main site (or part of the contribute page) so that people can email the most appropriate editor directly. That would be mean having a broad list of topics represented, including topics we cover already and ones that we don't but would like to have. 

I think it would be best to keep our editorial interests together rather than dispersed across our profiles at least for now as we're tweaking them, since it makes seeing and editing the bigger picture more difficult.

Edit away!
